mavlink-core/src/utils.rs

```diff
@@ -114,3 +110,13 @@ impl RustDefault for f64 {
         0.0
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_remove_trailing_zeroes_empty_slice() {
+        remove_trailing_zeroes(&[0]);
```

Comment:
pretty sure this should be `remove_trailing_zeroes(&[]);` instead.

Reply:
Yeah, I forgot to remove `0` after debugging with `0` :)